files/Trimble_Tekla/Sprint_2_1.ifc

```diff
@@ -0,0 +1,113 @@
+#59= IFCTUNNEL('2$M74xvK5FOvGEnGoq1S65',#9,'Tunnel name',$,$,#44,$,$,.ELEMENT.,.NOTDEFINED.);
+#60= IFCTUNNELPART('20vTfIXYfAPevvWLwb8hFH',#9,'TunnelPart name',$,$,#44,$,$,.ELEMENT.,.NOTDEFINED.,.NOTDEFINED.);
+#61= IFCLOCALPLACEMENT(#44,#43);
+#62= IFCALIGNMENT('2Sxv5OSKnEWR02TrT$AMeC',#9,'Name',$,$,#61,$,$);
+#63= IFCLOCALPLACEMENT(#61,#43);
+#64= IFCCARTESIANPOINTLIST2D(((0.,0.),(30.,0.),(45.,-15.),(60.,0.),(45.,15.)),$);
+#65= IFCINDEXEDPOLYCURVE(#64,(IFCLINEINDEX((1,2)),IFCARCINDEX((2,3,4)),IFCARCINDEX((4,5,2))),.F.);
+#66= IFCGEOMETRICREPRESENTATIONSUBCONTEXT('Axis','Model',*,*,*,*,#24,$,.MODEL_VIEW.,$);
+#67= IFCSHAPEREPRESENTATION(#66,'Axis','Curve2D',(#65));
+#68= IFCPRODUCTDEFINITIONSHAPE($,$,(#67));
+#69= IFCALIGNMENTHORIZONTAL('2TWCXVWkn9VBazjeWBGrnd',#9,'Name',$,$,#63,#68);
+#70= IFCCARTESIANPOINTLIST3D(((0.,0.,0.),(30.,0.,-1.),(45.,-15.,-2.),(60.,0.,0.),(45.,15.,2.),(30.,0.,1.)),$);
+#71= IFCINDEXEDPOLYCURVE(#70,(IFCLINEINDEX((1,2)),IFCARCINDEX((2,3,4)),IFCARCINDEX((4,5,6))),.F.);
+#72= IFCSHAPEREPRESENTATION(#66,'Axis','Curve3D',(#71));
+#73= IFCPRODUCTDEFINITIONSHAPE($,$,(#72));
+#74= IFCALIGNMENTVERTICAL('0WMb_$AsrAcuy2SFgkyJSl',#9,'Name',$,$,#63,#73);
+#75= IFCCARTESIANPOINT((0.,0.));
+#76= IFCALIGNMENTHORIZONTALSEGMENT('StartTag','EndTag',#75,0.,0.,0.,30.,$,.LINE.);
+#77= IFCALIGNMENTSEGMENT('3qU5S3vIb5juWGExtWnbKJ',#9,'Name','Description',$,$,$,#76);
+#78= IFCCARTESIANPOINT((30.,0.));
+#79= IFCALIGNMENTHORIZONTALSEGMENT('StartTag','EndTag',#78,4.71238898038469,15.,15.,47.1238898038469,$,.CIRCULARARC.);
+#80= IFCALIGNMENTSEGMENT('3$NmPu6vr4QAdmyZ64YgBt',#9,'Name','Description',$,$,$,#79);
+#81= IFCCARTESIANPOINT((60.,0.));
+#82= IFCALIGNMENTHORIZONTALSEGMENT('StartTag','EndTag',#81,1.5707963267949,15.,15.,47.1238898038469,$,.CIRCULARARC.);
+#83= IFCALIGNMENTSEGMENT('2ElVN5C3H06h2W4iYtI3jw',#9,'Name','Description',$,$,$,#82);
+#84= IFCALIGNMENTVERTICALSEGMENT('StartTag','EndTag',0.,30.,0.,-0.0333333333333333,-0.0333333333333333,$,.CONSTANTGRADIENT.);
+#85= IFCALIGNMENTSEGMENT('1tJqJ87b19L99czG5LwRhm',#9,'Name','Description',$,$,$,#84);
+#86= IFCALIGNMENTVERTICALSEGMENT('StartTag','EndTag',30.,47.1238898038469,-1.,-0.0424413181578388,0.0848826363156775,-15.0084776206116,.CIRCULARARC.);
+#87= IFCALIGNMENTSEGMENT('0G$o6JD1rDCPFnXDt7RhfU',#9,'Name','Description',$,$,$,#86);
+#88= IFCALIGNMENTVERTICALSEGMENT('StartTag','EndTag',77.1238898038469,47.1238898038469,0.,0.0848826363156776,-0.0424413181578387,-15.0084776206116,.CIRCULARARC.);
+#89= IFCALIGNMENTSEGMENT('1AyvfgsIf8y8wUF$yVH1k8',#9,'Name','Description',$,$,$,#88);
+#90= IFCREFERENT('1FkSJ8fDH2L9CB7ae0qf6n',#9,'Referent name',$,$,#44,$,.BOUNDARY.);
+#91= IFCGROUP('31ITpmYYT8fviPNXMEJDmg',#9,'Referent Group',$,$);
+#92= IFCRELASSIGNSTOGROUP('2ihJLXDhf8YOzV2GJRj7VS',#9,$,$,(#90),$,#91);
+#93= IFCRELAGGREGATES('1dlaXiYlX6CeNyyfoLjZPK',#9,$,$,#37,(#45));
+#94= IFCRELAGGREGATES('0MERbC7cH5$wxGxqLv9WM$',#9,$,$,#45,(#59,#62));

```

Comment:
Aggregating `IfcTunnel` and `IfcAlignment` should be two separate relationships.

Reply:
I don't like it if both of them have the same relationship with the same parent why should the relations be separated? Downstream applications have to read all the relationships anyway.